Week3/homework/step2-2.js

```diff
@@ -2,22 +2,32 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
-
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = startIndex; i <= stopIndex; i++) {
+    numbers.push(i);
+    if (i % 3 === 0) {
+      threeCallback();
+    }
+    if (i % 5 === 0) {
+      fiveCallback();
+    }
+    if (i % 3 === 0 && i % 5 === 0) {
+      threeCallback(fiveCallback());
```

Comment:
They shouldn't be called within each other, but separately

Reply:
@grod220 I called them outside the forloop but inside the function.

Reply:
Not sure what you mean. They should look like:
```
threeCallback(i)
fiveCallback(i)
```
See what I mean?

Reply:
function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
  const numbers = [];
  for (let i = startIndex; i <= stopIndex; i++) {
    numbers.push(i);
    if (i % 3 === 0) {

    }
    if (i % 5 === 0) {

    }
    if (i % 3 === 0 && i % 5 === 0) {

    }
  }
  threeCallback(fiveCallback());
  fiveCallback(i);
  threeCallback(i);
}

like this?